apps/src/gamelab/animationLibrary.json

```diff
@@ -350,16 +350,15 @@
     "bronze_1": {
       "name": "bronze_1",
       "aliases": [
-        "jumper",
```

Comment:
They're fairly easy to add - you just update the metadata file on S3 and next time someone updates the manifest the new aliases will be there.